packages/odyssey-react/.storybook/components/Markdown.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Note: There's no header on this because the lint rule doesn't care about contents the contents of hidden directories. Is this cool?